compliance/Azure_long_stopped_instances/README.md

```diff
@@ -0,0 +1,25 @@
+## Long-Stopped Instances
+ 
+### What it does
```

Comment:
it should mention that this works only with azure so the user doesn't need to scroll to the bottom.